src/jdk.jfr/share/conf/jfr/default.jfc

```diff
@@ -775,6 +775,11 @@
       <setting name="threshold">0 ms</setting>
     </event>
 
+    <event name="jdk.ZThreadEvent">
```

Comment:
This is just a question - do we want/need to have this event enabled for the 'low overhead profiling' setup?

Reply:
Yes. I always use default.jfc. The profile.jfc is creating too many allocation related events to make it usable when running with ZGC. I know that there has been some plans to fix that, but I don't know if that made it into JDK 17 or not.